charts/core-dump-handler/README.md

```diff
@@ -228,3 +228,38 @@ Daemonset
 * s3BucketName : Maps to the S3_BUCKET_NAME enviroment variable
 * 3Region : Maps to the S3_REGION enviroment variable
 * extraEnvVars: Option for passing additional configuration to the agent such as endpoint properties.
+
+### EKS setup for gitops pipelines (`eksctl` or similar)
+
+Set up a service account with a role that has access to S3 bucket (in `cluster.yanl`):
+
+```yaml
+iam:
+  withOIDC: true
+  serviceAccounts:
+    - metadata:
+      name: core-dump-admin
+      namespace: core-dump
+    attachPolicyARNs:
+      - arn:aws:iam::123456789011:policy/s3-write-policy
+```
+
+**Note**: here the namespace is `core-dump`, change it to the namespace where you installed the chart
+
+Example S3 policy:
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
+      "Resource": [
+        "arn:aws:s3:::my-core-dump-bucket",
+        "arn:aws:s3:::my-core-dump-bucket/*"
+      ]
+    }
+  ]
+}
+```
```

Comment:
Can you also add a sample command you used to install the chart.
Want to capture how it works with the schema validation

Reply:
I'm not using the chart.
It doesn't fit our gitops-only paradigm.
I render the chart offline using `helm template` and then install it through `fluxcd2`